interview-questions.md

```diff
@@ -7,44 +7,91 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: the foreign key would be Animal ID. It would be part of the sightings model.
 
   Researched answer:
 
+  When you declare that one model belonds to another, you instruct rails to maintain primary-key foreign-key info between the two instances of models. Rails supports 6 kinds of association:
+        belongs_to
+        has_one
+        has_many
+        has_many :through
+        has_one :through
+        has_and_belongs_to_many
+
 
 
 2. Which RESTful routes must always be passed params? Why?
```

Comment:
Update, delete, show and edit are always needed. great  job getting on the first try.